app/assets/stylesheets/styles/_footer.scss

```diff
@@ -0,0 +1,27 @@
+footer.page-footer {
+  bottom: 0;
```

Comment:
Properties should be ordered background-color, bottom, box-shadow, color, margin-bottom, width